Remotion/Core/UnitTests/Core.UnitTests.csproj

```diff
@@ -149,6 +149,9 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="..\..\..\SharedSource\Core\RoslynAnnotations\NullableAnnotations.cs">

```

Comment:
since we already have the NA included in Core.Core and made public, the include in projects that depend on Core.Core becomes redundant and possibly compiler-annoying.